projects/gateway-utilities/com.inductiveautomation.perspective/views/Utilities/Translation Management Page/view.json

```diff
@@ -0,0 +1,382 @@
+                },
+                "max": 100,
+                "min": 0,
+                "track": {
+                  "color": "",
+                  "style": {
+                    "classes": ""
+                  }
+                },
+                "value": {
+                  "enabled": true,
+                  "format": "0,0.##",
+                  "justify": "center",
+                  "style": {
+                    "classes": ""
+                  }
+                }
+              },
+              "render": "auto",
+              "resizable": true,
+              "sort": "none",
+              "sortable": false,
+              "strictWidth": false,
+              "style": {
+                "classes": ""
+              },
+              "toggleSwitch": {
+                "color": {
+                  "selected": "",
+                  "unselected": ""
+                }
+              },
+              "viewParams": {},
+              "viewPath": "",
+              "visible": true,
+              "width": ""
+            }
+          ],
+          "pager": {
+            "bottom": false,
+            "initialOption": 300,
+            "options": [
+              5,
+              10,
+              25,
+              50,
+              300
+            ]
+          },
+          "selection": {
+            "enableRowSelection": false
+          }
+        },
+        "type": "ia.display.table"
+      },
+      {
+        "children": [
+          {
+            "children": [
+              {
+                "events": {
+                  "dom": {
+                    "onClick": {
+                      "config": {
+                        "script": "\tViews.TranslationManagementPage.import_template(self,event)\n\t\t\n\t\t"
+                      },
+                      "scope": "G",
+                      "type": "script"
+                    }
+                  }
+                },
+                "meta": {
+                  "name": "importFile"
+                },
+                "position": {
+                  "basis": "213px"
+                },
+                "props": {
+                  "fileUploadIcon": {
+                    "color": "#FFFFFF",
+                    "name": "name",
+                    "style": {
+                      "background-color": "--callToAction"
+                    }
+                  },
+                  "style": {
+                    "background-color": "--callToAction",
+                    "classes": ""
+                  },
+                  "text-style": {
+                    "classes": ""
+                  }
+                },
+                "type": "ia.input.fileupload"
+              },
+              {
+                "events": {
+                  "component": {
+                    "onActionPerformed": {
+                      "config": {
+                        "script": "\tViews.TranslationManagementPage.download_template_file(self.view.custom.data)"
```

Comment:
Same thing as above. After moving python script to the General folder, this binding needs to be "General.TranslationManagementPage.download_template_file"